scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -47,7 +47,7 @@ public static BuildTargetResult Publish(BuildTargetContext c)
         [Target(
             nameof(PublishTargets.PublishVersionBadge),
             nameof(PublishTargets.PublishSdkInstallerFile),
-            //nameof(PublishTargets.PublishDebFileToDebianRepo),
+            nameof(PublishTargets.PublishDebFilesToDebianRepo),
```

Comment:
looked more into this, I don't think this function should get run on ubuntu, but its name is somewhat misleading. I will file and take an issue to clean up this file.